data/transport missions.txt

```diff
@@ -1412,6 +1412,137 @@ mission "Hallucination"
 
 
 
+mission "Shady passenger transport 1"
+	name "Shady passenger transport"
+	description "Discreetly transport a passenger to <destination>. If you are caught with him, you may be fined."
+	minor
+	repeat
+	source
+		attributes "near earth" core urban
+	destination
+		attributes frontier "dirt belt" rim north south pirate
+		distance 2 300
+	passengers 1
+		illegal 75000 `Your passenger turns out to be a small-time wanted criminal. As he is hauled off in chains, you vehemently protest that you knew nothing of his shady history. The authorities threaten you with jail time, and you quickly contact a lawyer. You wind up quite a bit poorer through a combination of fines, bribes, and attorney's fees (the breakdown remains somewhat unclear).`
+
+	to offer
+		random < 10
+		"rejected illegal jobs" < 3
+		"rejected illegal jobs" > -4
+
+	on offer
+		dialog `While having a drink at one of one of the spaceport's bars, a somewhat scruffy-looking man in a dark cloak pulls you aside and offers <payment> for discreet, no-questions-asked passage to <destination>, which is quite close by. This could be a quick and lucrative job. (it also sounds illegal, so you'll need to avoid getting scanned or landing on planets with high security.)`
+
+	on accept
+		"rejected illegal jobs" --
```

Comment:
rather than generic, this could be a positively valued condition like `getaway driver`

Reply:
Ooh, I love that.

Reply:
I figured you would :)

With #3128 you'd be able to make random "are you an evil dude" checks by just summing them and comparing to your criteria, while also requiring some minimum level of badassery:
```
to offer
    -1 * "said no to drugs" + "getaway driver" > 10
    "said no to drugs" < 0
    "getaway driver" < 0
```
and so on.